### PR TITLE
Added instruction to README.md

### PR DESCRIPTION
**Pull request by ludatkach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-# CCA20609
\ No newline at end of file
+# CCA20609
+Instruction
+1. Before starting work in CCA20609  on Webstorm:
+
+    1.1.  VCS => Git => Pull (pull updated Project)
+  
+     1.2. Create new branch [lastnameData] example: ivanov0617
```
Comment (Alex-Ashir) [inline README.md:7]: 1.2. Create a new branch [lastnameData] example: ivanov0617

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+  
+ 2. Start to work in new branch:
```
Comment (Alex-Ashir) [inline README.md:9]: 2. Start to work in a new branch:

File: README.md [hunk 3]
```diff
@@ -1 +1,35 @@
...
+     
+     4.2. Go to WebStorm => you are on your branch now
+     
+    4.3. On Local Branches => checkout to master
+    
+    4.4. On Local Branches => delete your branch
+    
+    4.5. Pull updated project to see your last changes
+    
+Repeat from beginning of Instruction every time.
```
Comment (Alex-Ashir) [inline README.md:35]: Repeat from the beginning of the instruction every time.

Comment (ludatkach) [reply README.md:35]: done

File: README.md (hunk 2 above)
Comment (ludatkach) [reply README.md:9]: done

**Pull request merged**